app/views/layouts/application.html.haml

```diff
@@ -17,6 +17,7 @@
     %link{ href: asset_pack_path('media/images/govuk-apple-touch-icon-167x167.png'), rel: "apple-touch-icon", sizes: "167x167" }/
     %link{ href: asset_pack_path('media/images/govuk-apple-touch-icon-152x152.png'), rel: "apple-touch-icon", sizes: "152x152" }/
     %link{ href: asset_pack_path('media/images/govuk-apple-touch-icon.png'), rel: "apple-touch-icon" }/
+    %link{ href: "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css", rel: "stylesheet" }/
```

Comment:
Just curious what this is being used for?

Reply:
i just commented about it on connors wip PR https://github.com/DFE-Digital/teacher-vacancy-service/pull/1758

Reply:
its amongst other things an svg icon library that is quite well used across the web